src/libs/utils.js

```diff
@@ -31,41 +31,29 @@ export const atom = initialValue => {
   }
 }
 
+/**
+ * Hook that returns the value of a given atom. When the atom changes, the component will re-render
+ */
+export const useAtom = theAtom => {
+  const [value, setValue] = useState(theAtom.get())
+  const handleChange = useCallback(v => setValue(v), [setValue])
```

Comment:
Why do this instead of just subscribing with `setValue` directly?

Reply:
Good question. It does work, but the atom subscription sends a second parameter, which React thinks might be an erroneous attempt to use a callback param like with `setState`, and it prints a warning in the console.

Reply:
_.ary maybe?

Reply:
Interesting. My gut tells me that's less clear. Thoughts?

Reply:
I think in this case it's more straightforward than having to pull in a whole other hook.

Reply:
Ah, the purpose of `useCallback` is to memoize the function because it's a dependency of the effect below. We need that regardless. The only thing `_.ary(1, setValue)` could replace is `v => setValue(v)`